profiles/seccomp/default.json

```diff
@@ -823,4 +825,4 @@
 			"excludes": {}
 		}
 	]
-}
\ No newline at end of file
+}
```

Comment:
I suspect this is what CI is failing on (while files _should_ have a newline at the end, the _generated_ version possibly doesn't have one);

```

[2021-02-26T02:03:34.884Z]  M profiles/seccomp/default.json
[2021-02-26T02:03:34.884Z] 
[2021-02-26T02:03:34.884Z] Please re-run go generate ./profiles/seccomp/
```